From 8b62f5c28119ccaf0d30f71d433de3a8b22be9a3 Mon Sep 17 00:00:00 2001
From: Iain Sandoe <iain@sandoe.co.uk>
Date: Mon, 18 Jan 2021 20:07:11 +0000
Subject: [PATCH] Darwin : Use a reserved name for the exception tables sect
 start.

We were using "GCC_except_table". Use "_GCC_except_table"
instead.
---
 gcc/config/darwin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git gcc/config/darwin.c gcc/config/darwin.c
index 8d86f5e8398..91c2f54ca16 100644
--- gcc/config/darwin.c
+++ gcc/config/darwin.c
@@ -2234,7 +2234,7 @@ darwin_emit_except_table_label (FILE *file)
 {
   char section_start_label[30];
 
-  ASM_GENERATE_INTERNAL_LABEL (section_start_label, "GCC_except_table",
+  ASM_GENERATE_INTERNAL_LABEL (section_start_label, "_GCC_except_table",
 			       except_table_label_num++);
   ASM_OUTPUT_LABEL (file, section_start_label);
 }
-- 
2.40.1

